lms/static/coffee/src/calculator.coffee

```diff
@@ -51,14 +51,17 @@ class @Calculator
     $calcWrapper = $('#calculator_wrapper')
     text = gettext('Open Calculator')
     isExpanded = false
+    icon = 'fa-calculator'
 
     $('div.calc-main').toggleClass 'open'
     if $calc.hasClass('closed')
+      icon = 'fa-calculator'
```

Comment:
Maybe I'm missing something, but this line seems redundant since `icon = 'fa-calculator'` is set 3 lines above. Is it really needed here?